packages/dom/src/core/sensors/pointer/PointerSensor.ts

```diff
@@ -234,6 +273,10 @@ export class PointerSensor extends Sensor<
   }
 
   private handlePointerUp(event: PointerEvent) {
+    if (!this.source) {
+      return;
+    }
```

Comment:
Now we're using global event listeners, we need to check if the user is dragging before stopping propagation (see https://github.com/measuredco/puck/issues/769).